parsely/src/main/java/com/parsely/parselyandroid/ParselyAPIConnection.kt

```diff
@@ -0,0 +1,62 @@
+/*
+    Copyright 2016 Parse.ly, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+package com.parsely.parselyandroid
+
+import android.os.AsyncTask
```

Comment:
Minor (🔍): Consider suppressing this deprecation as well; needs to be done on the file level (above the package declaration):

`@file:Suppress("DEPRECATION")`